### DISCUSSION TITLE
登录阿里云显示涉嫌违规，这是什么情况

### DISCUSSION THREAD

**Discussion by brianler** [AlistGo/alist#3733] [Q&A]
登陆后显示以下信息：
尊敬的用户你好，该账户因涉嫌违规，暂时无法使用阿里云盘服务，如有问题请联系客服邮箱：adrive@service.alibaba.com

**Comment by ilihu** [comment]
那你用官方客户端登录阿里云盘看看呢，如果官方客户端登录没问题，那才可能是alist的问题。否则，你应该联系阿里云的客服解封
